DEVELOPMENT_CONTRIBUTION.md

```diff
@@ -97,29 +97,30 @@ Open-source for web UI still feels like the Wild West, so we're trying to take a
 For your commit message, please use the following format:
 
 ```
-[<type>] <title>
-<BLANK LINE>
-<detailed commit message>
-<BLANK LINE>
-<reference to closing an issue>
-<BLANK LINE>
+<type>(optional scope): <description>
+
```

Comment:
Can we add back the < BLANK LINE >'s so it is explicit.

Reply:
Will be also nice to document how to put BREAKING CHANGE information

Reply:
Do you mean more details like this? Or, just linking to the existing info? 
https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#footer

Reply:
My personal feeling is to have in this document (maybe also copying it from the angular document). BUt also having an external link would be OK

Reply:
I think the link would be better. Otherwise we would have to keep the duplicate content in sync with the original.

Reply:
works for me, and you can anyway add this later if you wanna do something different than the Angular team is doing